docs/sources/whatsnew/whats-new-in-v9-4.md

```diff
@@ -296,6 +296,10 @@ We've added support for JWT authentication.
 
 We've added support for custom session parameters.
 
+## Postgres, MySQL, and MSSQL Datasources
+
+We've moved the `database` property under the `jsonData` key in the datasource configuration. This change is backwards compatible, and existing configurations will continue to work.
```

Comment:
Moved it from where? I'm unsure if the database property was located under jsonData, or if jsonData is the new location.

Reply:
jsonData is the new location.